### ISSUE TITLE
Dirt 2 very dark

### ISSUE THREAD

**Issue by jordanpucela** [fholger/openvr_fsr#10]
I have tried the mod in Dirt 2 and it works but the dashboard of the car leaves it very dark and I have tried various sharpness values ​​but nothing varies. Without the same car and circuit mod, the dashboard looks much brighter and looks perfectly. Do you know why it can be?

**Comment by fholger** [comment]
You could retry with the latest release, it might have fixed the issue. I can't test it right now, I'm afraid.

**Comment by DJSuah** [comment]
> I have tried the mod in Dirt 2 and it works but the dashboard of the car leaves it very dark and I have tried various sharpness values ​​but nothing varies. Without the same car and circuit mod, the dashboard looks much brighter and looks perfectly. Do you know why it can be?

Go to screen brightness settings! :D Works for me

**Comment by DJSuah** [comment]
> I have tried the mod in Dirt 2 and it works but the dashboard of the car leaves it very dark and I have tried various sharpness values ​​but nothing varies. Without the same car and circuit mod, the dashboard looks much brighter and looks perfectly. Do you know why it can be?


If you refer Dirt Rally 2 go to brighness settings and raise levels :D Works for me

**Comment by jordanpucela** [comment]
> > Probé el mod en Dirt 2 y funciona pero el tablero del auto lo deja muy oscuro y probé varios valores de nitidez pero nada varía. Sin el mismo modelo de automóvil y circuito, el tablero se ve mucho más brillante y se ve perfecto. ¿Sabes por qué puede ser?
> 
> Si te refieres a Dirt Rally 2, ve a la configuración de brillo y sube los niveles: D Me funciona

But without the mod the brightness is perfect and it is also especially inside the car.  Let's see if today I try the new version of the mod and I tell you.

**Comment by DJSuah** [comment]
> > > Probé el mod en Dirt 2 y funciona pero el tablero del auto lo deja muy oscuro y probé varios valores de nitidez pero nada varía. Sin el mismo modelo de automóvil y circuito, el tablero se ve mucho más brillante y se ve perfecto. ¿Sabes por qué puede ser?
> > 
> > 
> > Si te refieres a Dirt Rally 2, ve a la configuración de brillo y sube los niveles: D Me funciona
> 
> But without the mod the brightness is perfect and it is also especially inside the car. Let's see if today I try the new version of the mod and I tell you.

Me too! But after apply de FSR mod go to settings and raise levels.

**Comment by fholger** [comment]
Can anyone confirm if the latest version fixes the colors in Dirt 2?

**Comment by jordanpucela** [comment]
Tomorrow to see if I can try it and put it here

**Comment by DJSuah** [comment]
https://youtu.be/yCQuD2NIumE
This guy adjust the brightness and fix this problem like me. 
I see the new version cr2.
I haven't tried it yet. I'm looking forward to trying the new features. 

I'm very happy with the performance with any games with GTX 1070 / Samsung Odyssey and FSR mod Alpha 1.2.
No Man's Sky now is fully playable with FSR with ASW Auto setting and ASW enabled in cockpit. The gameplay now is very stable. 
The Forest VR now is perfect stable. 
Always use FSR Ultra Quality 0.77

Great work Fholger.

**Comment by jordanpucela** [comment]
> ¿Alguien puede confirmar si la última versión corrige los colores en Dirt 2?

Well, in Dirt2 you have to keep raising the brightness because otherwise it looks very dark, with the new version the same. But the brightness is turned up and voila.

**Comment by fholger** [comment]
This is now fixed in v1.0.

**Comment by jacklul** [comment]
> No Man's Sky now is fully playable with FSR with ASW Auto setting and ASW enabled in cockpit. The gameplay now is very stable.

Are you sure about it? It scales the resolution but the sharpening does not happen for me.
You can also notice when turning debug mode on it does not render on the screen.

This is how log looks like:

```
Initializing hooks...
Requested interface IVRSystem_022
Injecting GetRecommendedRenderTargetSize into IVRSystem_022
Requested interface IVRCompositor_027
Injecting Submit into IVRCompositor_027
Requested interface IVROverlay_025
Requested interface IVRChaperone_004
Error when submitting for eye 0: 101
Error when submitting for eye 1: 101
Requested interface IVRInput_010
Shutting down hooks...
```
